protocol/rpcconsumer/rpcconsumer.go

```diff
@@ -97,12 +97,12 @@ type RPCConsumer struct {
 }
 
 // spawns a new RPCConsumer server with all it's processes and internals ready for communications
-func (rpcc *RPCConsumer) Start(ctx context.Context, txFactory tx.Factory, clientCtx client.Context, rpcEndpoints []*lavasession.RPCEndpoint, requiredResponses int, cache *performance.Cache, strategy provideroptimizer.Strategy, metricsListenAddress string, maxConcurrentProviders uint, usageServerAddr string) (err error) {
+func (rpcc *RPCConsumer) Start(ctx context.Context, txFactory tx.Factory, clientCtx client.Context, rpcEndpoints []*lavasession.RPCEndpoint, requiredResponses int, cache *performance.Cache, strategy provideroptimizer.Strategy, metricsListenAddress string, maxConcurrentProviders uint, relayServerAddr string) (err error) {
```

Comment:
lets create a struct that passes both relay server address and metrics listen address ? will be more readable and easier to add more stuff in the future 

```go
type AnalyticsListenAddressess struct {
    MetricsListenAddress string 
    RelayServerAddress string
}
```